applications/GeoMechanicsApplication/custom_constitutive/windows.hpp

```diff
@@ -0,0 +1,8 @@
+// including windows.h adds some undesirable macros. Here, min and max macros are deactivated.
+#ifdef NOMINMAX
+#include <windows.h>
+#else
+#define NOMINMAX
+#include <windows.h>
+#undef NOMINMAX
+#endif
```

Comment:
sorry for asking again, but did you try removing this and directly including `#include <windows.h>`?

Since the include is now in a cpp I think it would work

Reply:
I replaced windows.hpp with windows.h and removed windows.hpp.